index.html

```diff
@@ -0,0 +1,251 @@
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+            <div class="item_content">
+                <a href="#"><img src="./assets/images/program_icon_05 1.svg" alt="program-icon"></a>
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from 
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+        </article>
+        <article class="program_footer">
+            <h5>SEE THE WHOLE PROGRAM</h5>
+        </article>
+      </article>  
+    </section>
+
+    <section class="speakers_container">
+        <article class="container speaker_holder">
+            <h2 class="speaker_header h-title" >
+                Featured Speakers
+                <a href="#" class="text-border"></a>
+
+            </h2>
+            <ul class="speakers">
+                <li class="speaker-box">
+                    <div class=" speaker-box_img"><img src="./assets/images/barack Obama.png" alt="speaker-image"></div>
+                    <div class="speaker-box_content">
+                        <h2 class="speaker-name">Barack Obama</h2>
+                        <p class="speaker-background">
+                            Berkman Professor of Entrepreneurial 
+                            Legal Studies at Harvard Law School
+                        </p>
+                        <p class="speaker-description">
+                            As the main venue for new media art production in Korea,
+                             Nabi promotes cross-disciplinary collaboration 
+                             and understanding among science 
+                             technology, humanities, and the arts.
+                        </p>
+                    </div>
+                </li>
+        
+
+                <li class="speaker-box">
+                    <div class=" speaker-box_img"><img src="./assets/images/cvimage2.JPG" alt="speaker-image"></div>
+                    <div class="speaker-box_content">
+                        <h2 class="speaker-name">Barack Obama</h2>
+                        <p class="speaker-background">
+                            Berkman Professor of Entrepreneurial 
+                            Legal Studies at Harvard Law School
+                        </p>
+                        <p class="speaker-description">
+                            As the main venue for new media art production in Korea,
+                             Nabi promotes cross-disciplinary collaboration 
+                             and understanding among science 
+                             technology, humanities, and the arts.
+                        </p>
+                    </div>
+                </li>
+            </ul>
```

Comment:
- According to the project requirements, you are to ensure that the entire speaker section is rendered dynamically using JavaScript, please remove all HTML templates for this section and render using only Javascript 👍 👍. Please fix this 😄  
- Also, please ensure that the images for the individual speakers are styled according to the design template. I suggest making them a little bit bigger. Please fix this for both mobile and desktop version 👍 👍 
Your project:
<img width="506" alt="trergrk" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/8d9ece94-620b-4720-a369-e768d8d57b2a">

Design template:
<img width="474" alt="wfefwae" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/fbabdbc2-d5e2-48b5-95ca-5b3db51069a7">